crates/polars-arrow/src/compute/arity.rs

```diff
@@ -105,6 +105,7 @@ where
 {
     let mut mut_bitmap = MutableBitmap::with_capacity(array.len());
 
+    #[allow(clippy::manual_unwrap_or_default)]
```

Comment:
This lint is currently broken:
https://github.com/rust-lang/rust-clippy/issues/12580